invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less

```diff
@@ -152,3 +152,8 @@ pre {
 
   }
 }
+
+a:hover .icon,
```

Comment:
could you double-check that its not overriding any other icon class.

Reply:
should the red highlighting on hover only be in the footer element?

Reply:
yes.